doc/api/corepack.md

```diff
@@ -0,0 +1,96 @@
+# Corepack
+
+<!-- introduced_in=REPLACEME -->
+<!-- type=misc -->
+
+> Stability: 1 - Experimental
+
+_[Corepack][]_ is an experimental tool shipped with Node.js to help with
+managing versions of your package managers. It exposes binary proxies for
+[Yarn][] and [pnpm][] that, when called, locate the package managers configured
+for your local projects, transparently install them, and finally run them
+without requiring explicit user interaction.
+
+This feature allows you to make sure that everyone in your team is using
+exactly the package manager version you want them to without having to check
+them into your repository or managing complex migrations across developers'
+systems.
+
+## Workflows
+
+### Enabling the feature
+
+Due to its experimental status Corepack currently needs to be explicitly
+enabled to have any effect. To do that, first make sure that neither Yarn nor
+pnpm are installed (uninstall them if needed), then run:
```

Comment:
Why does it matter if they are installed? If there are documented side effects, this section should reference that.

Reply:
It'll overwrite the existing symlinks. Will add a note about this.

Reply:
`corepack enable` warns before doing that, right? If so, it's probably fine to remove that sentence altogether.

Reply:
It doesn't warn yet